recipes/galeodes/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "galeodes" %}
+{% set version = "0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/galeodes-{{ version }}.tar.gz
+  sha256: 92465cb8485165c2889f90d708617b1f505ba484859836cf6e19f9545dcbe97d
+
+build:
+  noarch: python
```

Comment:
Upstream uses `scripts` so this cannot be `noarch`. You can either patch it here (Preferred) or change it.

https://github.com/qeeqbox/galeodes/blob/90ca7192ec173749a75334d92615dade357e3ea7/setup.py#L19